### PR TITLE
Feature/cds 67 elastic sql migrations

### PR DESCRIPTION
**Pull request by vfarah-if**
Elastic model enhancement of all address area data . 

- Included several postcode sql migrations dealing with populating United States Areas area data based on a well formatted zip/postcode. 
- Included Canada area data migration.

### Checklist

* [x] If this is a releasable change, has a news fragment been added?

  <details>
  <summary>Explanation</summary>
  
  A news fragment is required for any releasable change (i.e. code that runs in or affects production) so that a corresponding changelog entry is added when releasing.
  
  Check [changelog/README.md](https://github.com/uktrade/data-hub-api/blob/master/changelog/README.md) for instructions.
  
  </details>
  
* [x] Has this branch been rebased on top of the current `develop` branch?

  <details>
  <summary>Explanation</summary>
  
  The branch should not be stale or have conflicts at the time reviews are requested.
  
  </details>

* [x] Is the CircleCI build passing?

### General points

<details>
<summary><strong>Other things to check</strong></summary><p></p>

* Make sure `fixtures/test_data.yaml` is maintained when updating models
* Consider the admin site when making changes to models
* Use select-/prefetch-related field lists in views and search apps, and update them when fields are added
* Make sure the README is updated e.g. when adding new environment variables

</details>

See [docs/CONTRIBUTING.md](https://github.com/uktrade/data-hub-api/blob/develop/docs/CONTRIBUTING.md) for more guidelines.

### CODE REVIEW

File: datahub/dataset/company/views.py [hunk 1]
```diff
@@ -49,6 +51,9 @@ def get_dataset(self):
             'registered_address_country__name',
             'registered_address_county',
             'registered_address_postcode',
+            'registered_address_area__name',
+            'registered_address_area',
+            'registered_address_county',
```
Comment (ccp92) [inline datahub/dataset/company/views.py:56]: You've added county twice here (it already exists on line 53)

File: datahub/search/company/models.py [hunk 2]
```diff
@@ -109,4 +109,6 @@ class Company(BaseESModel):
         'address.postcode.trigram',
         'registered_address.country.name.trigram',
         'registered_address.postcode.trigram',
+        'address.area.name.trigram',
+        'registered_address.area.name.trigram',
```
Comment (ccp92) [inline datahub/search/company/models.py:113]: Would be good to have these lines alongside their top level objects (i.e. put `address` with other `address` and `registered_address` with other `registered_address`

File: tests.sh [hunk 3]
```diff
@@ -1,2 +1,3 @@
 #!/bin/bash -xe
+# If you use --reuse-db you can speed up test runs by excluding migrations
```
Comment (ccp92) [inline tests.sh:2]: Is this comment still required?

**Comment by vfarah-if** [comment]
Split this PR out into 3 separate tickets

**Pull request closed without merging**